src/test/java/utils/example/utils/MyArrayListTest.java

```diff
@@ -11,21 +11,42 @@
     public void addListTest() {
         MyArrayList expectedEmployeeData = new MyArrayList();
         expectedEmployeeData.add("Muhammad");
-        expectedEmployeeData.add("Ahmed");
-        expectedEmployeeData.add("abdullah");
+        expectedEmployeeData.add(1);
+        expectedEmployeeData.add(33.5);
 
         MyArrayList employee = new MyArrayList();
         assertNotNull(expectedEmployeeData);
         assertEquals(employee.equals(expectedEmployeeData), expectedEmployeeData.equals(employee));
     }
 
     @Test
-    public void getIndexTest(){
+    public void getIndexTest() throws Exception {
```

Comment:
No Idea what is this test is doing. Its doing nothing. Create list -> Add and object into it -> Get same object from the list and then see if you get back the same object.